index.html

```diff
@@ -3989,22 +3989,20 @@ <h3>JSON-LD</h3>
 
         <p>
 As elaborated upon in Section <a href="#json-processing"></a>, some software
-applications might not perform full JSON-LD processing. In order to increase
-interoperability, <a>conforming document</a> authors are urged to not use
-JSON-LD features that are not easily detected when not performing full
-JSON-LD processing. This guidance includes:
+applications might not perform full JSON-LD processing. To increase
+interoperability, <a>conforming document</a> authors are urged to avoid using
+JSON-LD features that are easily detected only when performing full
+JSON-LD processing. These features include:
```

Comment:
```suggestion
applications might not perform dynamic JSON-LD processing, electing to
use static processing that will, for example, reject documents with unrecognized
contexts. To increase interoperability, <a>conforming document</a> authors are
urged to avoid using JSON-LD features that are easily detected only when
performing dynamic JSON-LD processing. <a>Conforming document</a> authors
are advised that usage of certain JSON-LD features might reduce 
interoperability. These features include:
```